docs/content/preview/architecture/system-catalog.md

```diff
@@ -32,90 +32,77 @@ To get the details of the names and type information of columns in a table, you
 
-The _pg_attribute_ table stores metadata about the columns (attributes) of all relations (tables, views, indexes, and so on) in the database. It is extensively used by the query optimizer to generate efficient execution plans, by the database engine to enforce data integrity rules, and by administrators for tasks such as database documentation, maintenance, and troubleshooting.
+Let us look at some of the important information that can be fetched using the system catalog tables and views followed by a summary of other members.
 
-## pg_class
+## Schema
 
-The _pg_class_ table serves as a comprehensive repository for storing metadata about all relations (tables, views, indexes, sequences, and other relation types) in the database. It is used to generate efficient execution plans, by storage managers to handle data files and disk usage, and by administrators for tasks such as monitoring, tuning, and maintaining the database.
+The schema details of the various database objects are stored in multiple tables as follows.
 
-## pg_constraint
+- _pg_database_ : stores the list of all the databases in the system
+- _pg_namespace_ : stores metadata about schemas, including schema names, owner information, and associated privileges.
+- _pg_class_ : stores metadata about all relations (tables, views, indexes, sequences, and other relation types) in the database.
+- _pg_attribute_ : stores information about the columns (attributes) of all relations (tables, views, indexes, and so on) in the database.
+- _pg_index_ : stores detailed metadata about indexes, including details such as the indexed columns, index types, and index properties like uniqueness and inclusion of nullable values.
+- _pg_constraint_ : stores information about constraints on tables. These can include unique constraints, check constraints, primary key constraints, and foreign key constraints.
 
-The _pg_constraint_ table stores information about constraints on tables. Constraints are rules that specify certain conditions that the data in a table must fulfill. These can include unique constraints, check constraints, primary key constraints, and foreign key constraints.
+This information is typically fetched using convenient views like:
 
-## pg_database
+- _pg_views_ : provides details on views and their definitions
+- _pg_tables_ : provides details on tables, their ownership and basic properties (eg, if the table has any indexes etc)
+- `information_schema.tables` : provides table information as per SQL standard.
+- `information_schema.columns` : provides column information as per SQL standard.
+- `information_schema.columns` : provides view information as per SQL standard.
```

Comment:
information_schema.views?

Reply:
@premkumr Did you miss this comment - is this correct? Why repeat columns?

Reply:
Yup. looks like I missed this comment :( . Will raise a PR soon ...